private-undistributed.yml

```diff
@@ -420,6 +420,7 @@ allow-licenses:
   - 'EPL-1.0'
   - 'EPL-2.0'
   - 'EPL-2.0 AND Apache-2.0 AND BSD-3-Clause'
+  - 'EPL-2.0 AND BSD-3-Clause AND Apache-1.1 AND EPL-2.0 AND EPL-1.0 AND Apache-1.1 AND Apache-2.0 AND BSD-2-Clause AND Apache-2.0 AND Apache-1.1 AND BSD-3-Clause'
```

Comment:
I'm realizing that this combination is redundant quite a bit, seems like it can be summarized as : 
```
EPL-2.0 AND BSD-3-Clause AND Apache-1.1 AND EPL-1.0 AND Apache-2.0 AND BSD-2-Clause
```

But the long string is what GitHub returns, so that's the one that need to be allowed unfortunately...